core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -50,6 +53,17 @@
  */
 public abstract class ServerCall<RespT> {
   /**
+   * {@link Attributes.Key} for the remote address of server call.
+   */
+  public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+          Attributes.Key.of("io.grpc.RemoteAddr");
```

Comment:
Hah, yes, totally! I was planning to send a follow up removing the namespace and updating docs for attributes because now, there's no point in making them namespaced. I guess I can do make this happen in this PR.